src/components/Message/Readme.md

```diff
@@ -11,7 +11,17 @@ Example 1:
   <MessageText>All good man! 😂</MessageText>
 </Message>
 <Message authorName="Joe" avatarUrl="https://randomuser.me/api/portraits/lego/6.jpg" isOwn={true} >
-  <MessageText>Look! it even detects links like this - https://developerzen.com - and email emails like this@gmail.com</MessageText>
+  <MessageText>Look! it even detects links like this - https://developerzen.com - and email addresses` like this@gmail.com</MessageText>
+</Message>
+<Message authorName="Jack" avatarUrl="https://randomuser.me/api/portraits/lego/2.jpg" isOwn={false} >
+  <MessageMedia title="New Hires" subtitle="Meet our new hires!">
+    <img src="https://images.unsplash.com/photo-1525422847952-7f91db09a364" />
+  </MessageMedia>
+  <MessageItems>
+    <MessageItem title="Annie Alexander" subtitle="Hello, I am new at this company" image="https://randomuser.me/api/portraits/women/28.jpg" />
```

Comment:
You should change this example to be a use case that would fit the UI a bit better.
This is not meant to be used to show user messages more more like a list of items - news items, product recommendations, ...